Orange/data/sql/backend/postgres.py

```diff
@@ -165,6 +165,8 @@ def _guess_variable(self, field_name, field_metadata, inspect_table):
         if type_code in CHAR_TYPES:
             if inspect_table:
                 values = self.get_distinct_values(field_name, inspect_table)
+                # remove trailing spaces
+                values = [v.rstrip() for v in values]
```

Comment:
Why not modify query to cast to ::text like it will be done in further selects?

Reply:
`get_distinct_values` encapsulates `filed_name` into quotes and the query becomes invalid. On request, I move the encapsulation from `get_distinct_values` and add outside function calls.

Reply:
no, it is ok the way it is.